src/utils/hashes/ledgerHash.ts

```diff
@@ -76,7 +76,6 @@ export function computeSignedTransactionHash(tx: Transaction | string): string {
   let txObject: Transaction
   if (typeof tx === 'string') {
     txBlob = tx
-    // TODO: type ripple-binary-codec with Transaction
```

Comment:
is there a reason we aren't leaving this TODO in here? Or at least creating an issue in r-b-c